ldms/python/ldmsd/ldmsd_yaml_parser

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import os, sys
+import yaml
+import errno
+import argparse
+from ldmsd.parser_util import *
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="LDMS Monitoring Cluster Configuration")
+    parser.add_argument("--ldms_config", metavar="FILE", required=True,
+                        help="The ldmsd load balance domain configuration file. "
```

Comment:
@nick-enoent I believe the help string of `--ldms-config` may confuse readers as this is in the `ldmsd` proper. It is the argument that receives the path to the YAML configuration file, isn't it? My point is that this has nothing to do with load balancing in Maestro.

Reply:
Good catch, thanks Mon! I updated this in [9b4113a](https://github.com/ovis-hpc/ldms/pull/1461/commits/9b4113a2e362def38eca1543eb71d2e326483e58)